kubernetes/operationcode_python_backend/overlays/prod/ingress.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    kubernetes.io/tls-acme: "true"
+  name: back-end
+spec:
+  rules:
+  - host: api.operationcode.org
+    http:
+      paths:
+      - backend:
+          serviceName: operationcode-backend-service
```

Comment:
this should be `back-end-service` if you want to receive traffic. However, we'll likely want to hold off on that or put this up on an alternate URL for testing until it works